draft-bbs-signatures.md

```diff
@@ -841,6 +845,132 @@ Procedure:
 4. return (a_octets || e_octets || s_octets)
 ```
 
+### OctetsToProof
+
+The `octets_to_proof` operations describes how to parse an octet string representation of a proof, validate it and return the underlying components.
```

Comment:
```suggestion
This operation describes how to decode an octet string representing a proof, validate it and return the underlying components that make up the proof.
```

Suggestion to make this more consistent language with how we introduce each operation
```suggestion
The `octets_to_proof` operations describes how to parse an octet string representation of a proof, validate it and return the underlying components.
```

Reply:
Thank you! Updated